src/stylesheets/elements/_buttons.scss

```diff
@@ -1,181 +1,215 @@
   &:focus {
     outline-offset: units(2px);
   }
+  &--disabled,
+  &:disabled {
+    box-shadow: none;
+    @include u-border("disabled");
+  }
 }
 
-// DEPRECATED: Use .sds-button.sds-button--circular
 .sds-button--circle {
   border-radius: 50%;
-  color: color('secondary-darker');
-  border: 0.65px solid darkgray;
-  height: 2rem;
-  width: 2rem;
-  padding: 2px 0 0 1px;
+  @include u-border("primary");
+  @include u-border-width("2px");
+  color: color("secondary-darker");
+  height: units(4);
+  width: units(4);
+  @include u-display("inline-flex");
+  @include u-flex("align-center");
+  @include u-flex("justify-center");
+  margin: unset;
+  padding: 0;
 
   .svg-inline--fa {
     align-self: center;
   }
 
-  &.usa-button--hover,
-  &:hover {
-    background-color: color('primary-light');
-    color: color('secondary-darker');
+  &.sds-button--small {
+    height: units(3);
+    width: units(3);
+    padding: 0;
+    font-size: size('body', 1);
   }
-
-  &.usa-button--active,
-  &:active {
-    background-color: color('secondary-darker');
-    color: color('white');
-  }
-
-  &.usa-focus,
-  &:focus {
-    outline-offset: units(2px);
+  &.sds-button--big {
+    height: units(5);
+    width: units(5);
+    font-size: size('body', 8);
   }
 }
 
 .usa-button--outline {
   background-color: transparent;
-  box-shadow: $button-stroke color('primary');
+  box-shadow: none;
 
   &:visited {
-    color: color('primary');
+    color: color("primary");
   }
 
   &.usa-button--hover,
   &:hover {
     background-color: transparent;
-    box-shadow: $button-stroke color('primary-light');
+    box-shadow: none;
   }
 
   &.usa-button--active,
   &:active {
     background-color: transparent;
-    color: color('secondary-darker');
-    box-shadow: $button-stroke color('secondary-darker');
+    color: color("secondary-darker");
+    box-shadow: none;
   }
 
   &.usa-button--inverse {
     background-color: transparent;
-    box-shadow: $button-stroke color('primary');
-    color: color('white');
+    color: color("white");
+    box-shadow: none;
 
     &:visited {
-      color: color('base-lighter');
+      color: color("base-lighter");
+      box-shadow: none;
     }
 
     &.usa-button--hover,
     &:hover {
       background-color: transparent;
-      box-shadow: $button-stroke color('base-lightest');
-      color: color('base-lightest');
+      color: color("base-lightest");
+      box-shadow: none;
     }
 
     &.usa-button--active,
     &:active {
       background-color: transparent;
-      box-shadow: $button-stroke color('white');
-      color: color('base-light');
+      color: color("base-light");
+      box-shadow: none;
+    }
+
+    &:disabled {
+      background-color: transparent;
+      color: color("base-light");
+      box-shadow: none;
     }
   }
 }
-
 .usa-button--base {
-  color: color('gold-70');
-  background-color: color('accent-warm-lighter');
-  box-shadow: $button-stroke color('base-light');
+  color: color("gold-70");
```

Comment:
the background should be our base color if we're calling it .usa-button--base. Use a different sds-button modifier or see if we can change our base color if we need this